index.html

```diff
@@ -0,0 +1,197 @@
+      <h2>Tonic</h2><br>
+      <p class="inline-paragraph" id="canopy">CANOPY</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">Back End Dev</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">2023</p>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li class="my-button">HTML</li>
+        <li class="my-button">CSS</li>
+        <li class="my-button js">Javascript</li><br><br>
+      </ul>
+      <button type="button" class="see-project">See Project</button>
+    </section>
+
+
+    <section>
+      <img src="img/multi2.svg" class="postsvg">
+      <h2>Multi-Post<br>
+        Stories</h2><br>
+      <p class="inline-paragraph" id="canopy">CANOPY</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">Back End Dev</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">2023</p>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li class="my-button">HTML</li>
+        <li class="my-button">CSS</li>
+        <li class="my-button js">Javascript</li><br><br>
+      </ul>
+      <button type="button" class="see-project">See Project</button>
+    </section>
+
+
+    <section class="aboutmyself">
+      <h2>About
+        Myself</h2>
+      <p> Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my
+        work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <img src="img/LET’S CONNECT.svg" alt="humburger"><br><br>
+    </section>
+
+
+    <section>
+      <ul class="social-icons">
+        <li>
+          <img src="img/twitter icon.svg">
+        </li>
+        <li>
+          <img src="img/Linkedin icon.svg">
+        </li>
+        <li>
+          <img src="img/Mobile icon.svg">
+        </li>
+        <li>
+          <img src="img/git icon.svg">
+        </li>
+        <li>
+          <img src="img/peace icon.svg">
+        </li><br>
+      </ul>
+      <button type="button" class="getmy">Get my resume</button>
+      <section class="this-site">
+        <div class="ingine">
+          <span class="text">
+            <h3>Languages</h3>
+          </span><img src="img/faced.svg">
+        </div>
+          <img src="img/javascript.svg">
+            <img src="img/html.svg">
+           <img src="img/css.svg">
+        <div class="unionsvg">
+          <span class="text">
+            <h3>Framework</h3>
+          </span>
+          <img src="img/unionforward.svg">
+          <br><br><br><br>
+        </div>
+        <div class="kesho">
+          <img src="img/Separator Bottom.svg">
+        </div>
+        <div class="leo">
+          <span class="text">
+            <h3>Skills</h3>
+          </span>
+          <img src="img/unionforward.svg">
+        </div>
+        <img src="img/Separator Bottom.svg">
```

Comment:
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") are meant to be in a `ul`, each of them in a `li `. :+1:  I'll suggest you update this. :smile: 
- [ ] Also, the  "JavasScript", "HTML" and "CSS" are meant to be a list `ul` and each of them a `li`, I suggest you look into this too. :smile: